README.md

```diff
@@ -132,6 +156,39 @@ TectonTriggerOperator(
 )
 ```
 
+### Feature Table Ingestion Materialization Job Submission
+```python
+import pandas as pd
+from datetime import datetime
+
+from airflow_tecton.operators.tecton_feature_table_trigger_operator import TectonFeatureTableTriggerOperator
+
+WORKSPACE = "prod"
+FEATURE_VIEW = "df_users"
+
+
+def generate_df():
+    data = {'name': ['Tom', 'Joseph', 'Krish', 'John'],
+            'age': [20, 21, 19, 18],
+            'ts': [datetime.fromtimestamp(1674819600),
+                   datetime.fromtimestamp(1675211580),
+                   datetime.fromtimestamp(1674347580),
+                   datetime.fromtimestamp(1674725580)]}
+    return pd.DataFrame(data)
+
+...
+
+tecton_trigger = TectonFeatureTableTriggerOperator(
+    task_id="trigger_tecton_feature_table",
+    workspace=WORKSPACE,
+    feature_view=FEATURE_VIEW,
+    online=True,
+    offline=True,
+    df_generator=generate_df,
```

Comment:
i think you should add dag=dag here to be clear

Reply:
prob goes for the original examples too

Reply:
sure